huey/contrib/djhuey/management/commands/run_huey.py

```diff
@@ -73,9 +72,6 @@ def handle(self, *args, **options):
         consumer_options.setdefault('verbose',
                                     consumer_options.pop('huey_verbose', None))
 
-        if not options.get('disable_autoload'):
```

Comment:
This makes the `--disable-autoload` do nothing, at current. Unfortunately the command-line option will be parsed after the `AppConfig.ready()` hook, so there's no way to retain the current behaviour. But that may be acceptable since it seems few people are likely to use the option? It was added in 401740f88d5a1ab3ea02eb2d79cff4c474dc671e with minimal documentation, following a discussion in #362.